epoxy-processor/src/main/java/com/airbnb/epoxy/GeneratedModelWriter.kt

```diff
@@ -174,6 +174,7 @@ internal class GeneratedModelWriter(
             builderHooks?.beforeFinalBuild(this)
 
             addSuperinterface(modelInterfaceWriter.writeInterface(info, this.build().methodSpecs))
+            addOriginatingElement(info.superClassElement)
```

Comment:
Shouldn't the originating element be the model class itself rather than its super?

Reply:
Yes. I was thrown off by this as well, but `superClassElement` in this case _is_ the original model class. The model, if I understood it correctly, is the superclass of the type this processor is generating.